notification-spring/src/test/java/org/duracloud/notification/SpringNotificationFactoryTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ *     http://duracloud.org/license/
+ */
+package org.duracloud.notification;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+/**
+ * @author Andrew Woods
+ * Date: 3/11/11
```

Comment:
Author and date here are probably not right

Reply:
Rectified !880fba5